test/fn-system-tests/exec_runner_status_test.go

```diff
@@ -405,3 +412,42 @@ func TestExampleLogStreamer(t *testing.T) {
 	}
 
 }
+
+// TestStatus_verifyIO verifies RunnerProtocol.Status2(..) behavior
+// Note: this test depends on the default status checker image
+// which echoes its input
+func TestStatus_verifyIO(t *testing.T) {
+	buf := setLogBuffer()
+	defer func() {
+		if t.Failed() {
+			t.Log(buf.String())
+		}
+	}()
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	// Connect to any node
+	r := runnerGrpcServerAddr(-1)
```

Comment:
for deterministic tests, should we just pick same node? or hit all?

Reply:
This is a very simple test so targeting all nodes seems wasteful.  I'll pick a node.